recipe/meta.yaml

```diff
@@ -354,51 +320,35 @@ outputs:
         - {{ cdt('xcb-util-wm-devel') }}          # [linux and not (aarch64 or x86_64)]
         - jom                                    # [win]
         - make  # [not win]
-        - python
-        - pyqt-builder
-        - dbus-python  # [unix]
         - pkg-config
-        - pyqt
-        - qt-main 5.15.*
-        - qt-webengine 5.15.*
-        - qtwebkit 5.*
-        - setuptools
-        - wheel
-        - pip
       host:
         - python
-        - pip
-        - setuptools
-        - wheel
-        - sip
-        - toml
-        - pyqt-builder
-        - packaging
+        - pyqt-builder >=1.9,<2
+        - sip >=5.3,<7
         - qt-main 5.15.*
-        - qt-webengine 5.15.*
-        - qtwebkit 5.*
-        - pyqt
+        - {{ pin_subpackage('pyqt', exact=True) }}
         - {{ cdt('mesa-libgl-devel') }}          # [linux]
         - {{ cdt('mesa-libegl-devel') }}         # [linux]
       run:
         - qt-main 5.15.*
-        - qtwebkit 5.*
-        - qt-webengine 5.15.*
         - python
         - {{ pin_subpackage('pyqt', max_pin='x.x') }}
-{% if enable_testingui %}
     test:
+{% if enable_testingui %}
       imports:
         - PyQt5.QtChart
 {% endif %}
+      requires:
+        - pip
+      commands:
+        - pip check
 
 about:
   home: https://www.riverbankcomputing.com/software/pyqt/
   license: GPL-3.0-only
   license_family: GPL
   license_file: pyqt/LICENSE
   summary: Python bindings for the Qt cross platform application toolkit
-  # The remaining entries in this section are optional, but recommended.
   description: |
     PyQt5 is a comprehensive set of Python bindings for Qt v5.
     It is implemented as more than 35 extension modules and enables Python
```

Comment:
`dev_url: https://www.riverbankcomputing.com/software/pyqt/`

Reply:
PKG_INFO says 
```
Support may be obtained from the PyQt mailing list at
https://www.riverbankcomputing.com/mailman/listinfo/pyqt/.
```
At least this way users get both links, what do you think?

Reply:
I would direct users to https://riverbankcomputing.com/support/pyqt to k now how to get support.